pipelines/jobs/configurations/jdk17u_pipeline_config.groovy

```diff
@@ -153,8 +153,18 @@ class Config17 {
                 buildArgs           : [
                         'temurin'   : '--create-jre-image --create-sbom'
                 ]
-        ]
+        ],
 
+        aarch64Windows: [
```

Comment:
Should this be added?

Reply:
if we do not have it enabled in jdk17u.groovy or jdk17u_prototype.groovy, it wont be used in the build

Reply:
@andrew-m-leonard if the question is "should Windows aarch64" be in the prototype pipeline then my answer is yes, please. I'm working on adding a second machine to the build farm this week to resolve the redundancy issues we had.